packages/twenty-front/src/modules/support/components/SupportChat.tsx

```diff
@@ -98,6 +101,8 @@ export const SupportChat = () => {
     currentWorkspaceMember,
   ]);
 
+  if (loading) return <StyledSupportChatSkeletonLoader />;
```

Comment:
not sure we need a SkeletonLoader for the support button, do we?

Reply:
The figma designs indicate so!